image_publisher/src/nodelet/image_publisher_nodelet.cpp

```diff
@@ -100,7 +100,22 @@ class ImagePublisherNodelet : public nodelet::Nodelet
       if (flip_image_)
         cv::flip(image_, image_, flip_value_);
 
-      sensor_msgs::ImagePtr out_img = cv_bridge::CvImage(std_msgs::Header(), "bgr8", image_).toImageMsg();
+      std::string encoding = "bgr8";
```

Comment:
instead of "bgr8", it should be sensor_msgs::image_encodings::BGR8 (yes, it was wrong before).